[I 10:00:05.897 NotebookApp] Serving notebooks from local directory: /gpfs/home4/zimmer/neutrino_clustering_V2
[I 10:00:05.898 NotebookApp] Jupyter Notebook 6.4.12 is running at:
[I 10:00:05.898 NotebookApp] http://localhost:5236/?token=b6dd048a5a108784e50e14e38507a4fa2613b6785c9eb777
[I 10:00:05.898 NotebookApp]  or http://127.0.0.1:5236/?token=b6dd048a5a108784e50e14e38507a4fa2613b6785c9eb777
[I 10:00:05.898 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 10:00:13.209 NotebookApp] 
    
    To access the notebook, open this file in a browser:
        file:///gpfs/home4/zimmer/.local/share/jupyter/runtime/nbserver-601789-open.html
    Or copy and paste one of these URLs:
        http://localhost:5236/?token=b6dd048a5a108784e50e14e38507a4fa2613b6785c9eb777
     or http://127.0.0.1:5236/?token=b6dd048a5a108784e50e14e38507a4fa2613b6785c9eb777
[I 10:00:56.232 NotebookApp] Creating new notebook in 
[I 10:01:06.298 NotebookApp] Kernel started: b4617537-19e2-4ed4-be38-905b5e759fb4, name: python3
[W 10:01:06.314 NotebookApp] delete /hackaton_gravity-jvsc-72dbd7a5-39dc-4fed-bd0c-8e5f9f298f2ed83b7139-3caf-4eba-81cc-84e38dec6921.ipynb
[W 10:01:06.315 NotebookApp] Skipping trash for /gpfs/home4/zimmer/neutrino_clustering_V2/hackaton_gravity-jvsc-72dbd7a5-39dc-4fed-bd0c-8e5f9f298f2ed83b7139-3caf-4eba-81cc-84e38dec6921.ipynb, [Errno 13] Permission denied: b'/gpfs/home4/.Trash-59104'
